### ISSUE TITLE
Graph multiple properties with identical names

### ISSUE THREAD

**Issue by BakuDev** [ARCANEDEV/SEO-Helper#31]
- SEO-Helper Version: Latest
- Laravel Version: 5.5.5
- PHP Version: 7.1

### Description:
It's impossible to add multiple properties to Graph with identical names.

### Steps To Reproduce:

```php
$this->seoGraph()->addProperty('locale:alternate', 'en');
$this->seoGraph()->addProperty('locale:alternate', 'ru');
```

Will output only last one in rendered HTML.

**Comment by arcanedev-maroc** [comment]
I've implemented two methods to manage the locales:

```php
// For `og:locale` property
$this->seoGraph()->setLocale('fr_FR');

// For `og:locale:alternate` property
$this->seoGraph()->setAlternativeLocales(['fr_FR', 'es_ES']); 
```

**Comment by BakuDev** [comment]
Hi there!

Cool! Thank you!
Please merge so we can update via composer.

**Comment by arcanedev-maroc** [comment]
Released: 1.4.1

**Comment by BakuDev** [comment]
Unfortunately, getting error after update to 1.4.1: 

Call to undefined method Arcanedev\SeoHelper\SeoOpenGraph::setLocale()
Same for setAlternativeLocales()

**Comment by BakuDev** [comment]
Updated to 1.4.1

**Comment by arcanedev-maroc** [comment]
My bad 😜: #33

**Comment by arcanedev-maroc** [comment]
Fixed: v1.4.2
